src/env.py

```diff
@@ -810,10 +827,9 @@ def cleanup_(restore_items: list):
                     f"Directory {src} is restored form {backup_dir}")
 
         elif type_ == "user":
-            username = item["username"]
-            run(["userdel", username, "-r"])
+            run(["userdel", src, "-r"])
             env_logger.debug(
-                f"User {username} is delete with it home directory")
+                f"User {src} is delete with it home directory")
```

Comment:
This is written wrong. Maybe just replace it with "User {src} has been removed"

Reply:
Makes sense. will fix